pkgs/tools/misc/hostmux/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, installShellFiles, openssh, tmux, ... }:
```

Do not use ellipsis.
```suggestion
{ lib
, stdenv
, fetchFromGitHub
, installShellFiles
, openssh
, tmux
}:
```